ietf-layer0-types-ext.yang

```diff
@@ -51,4 +51,494 @@ module ietf-layer0-types-ext {
    
    The typedefs and groupings will be added here. */
 
-}
+// identity
+
+    identity modulation {
+    description "base identity for modulation type";
+  }
+
+  identity QPSK {
+    base modulation;
+    description
+      "QPSK (Quadrature Phase Shift Keying) modulation";
+  }
+
+  identity DP-QPSK {
+    base modulation;
+    description
+      "DP-QPSK (Dual Polarization Quadrature
+       Phase Shift Keying) modulation";
+  }
+  identity QAM8 {

```

Comment:
Editorial: please add an empty line to improve readability

Reply:
ok, done